src/PointwiseFunctions/AnalyticData/GrMhd/RiemannProblem.hpp

```diff
@@ -0,0 +1,336 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <limits>
+#include <string>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/AnalyticData/AnalyticData.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Minkowski.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/IdealFluid.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/Hydro/TagsDeclarations.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+// IWYU pragma: no_include <pup.h>
+
+/// \cond
+namespace PUP {
+class er;  // IWYU pragma: keep
+}  // namespace PUP
+/// \endcond
+
+namespace grmhd::AnalyticData {
+/*!
+ * \brief Initial conditions for relativistic MHD Riemann problems
+ *
+ * The standard problems were first collected in \cite Balsara2001. A complete
+ * Riemann solver for the RMHD equations is presented in \cite Giacomazzo2006
+ * and can be downloaded from https://www.brunogiacomazzo.org/?page_id=395
+ *
+ * The domain is from \f$[-0.5,0.5]^3\f$ with periodic boundary conditions in
+ * \f$y\f$ and \f$z\f$. The problems are usually run at a resolution of 1600
```

Comment:
Say somewhere in the discussion of the domain that the discontinuity is initially at x = 0.